README.md

```diff
@@ -126,4 +126,20 @@ The [Node.js Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_O
 
 The [Node.js Moderation Policy](https://github.com/nodejs/admin/blob/master/Moderation-Policy.md) applies to this WG.
 
+# Code of Conduct
```

Comment:
these are duplicates of existing code of conduct and moderation policies already in the document

Reply:
Git sticky fingers on my side, will remove